deploy/tools/restart_proxy.sh

```diff
@@ -18,12 +18,13 @@ function env_vars {
 pushd "${DEPLOYDIR}"
 ls
 env_vars
-
+STRATOS_VERSION=v2.0.0-$(git log -1 --format="%h")
```

Comment:
Remove v

Reply:
We should read the 2.0.0 part from the package file, otherwise we'll need to update this everytime we release 2.0.1 etc.